staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -865,8 +865,12 @@ func growSlice(v reflect.Value, maxCapacity int, sizes ...int) {
 }
 
 // Watch implements storage.Interface.Watch.
+// TODO(#115478): for beta the WatchList feature also must/should support streaming.
```

Comment:
This TODO is somewhat misleading to me. Watch by definition supports streaming :)

We probably want to say sth like:
TODO(...): For beta of WatchList feature, etcd3 implemenation should always support it.

[(or sth like that)]

Reply:
done, reworded the comment.